### PR TITLE
MULE-11098: Change the TLS context of the WSC configuration for an Object that handles security.

### PR DESCRIPTION
**Pull request by juandesi**

### CODE REVIEW

File: extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  @Summary("The password used to access the private key.")
+  private String keyPassword;
+
+  @Parameter
+  @Summary("The location of the KeyStore file")
+  private String keyStorePath;
+
+  @Parameter
+  @Summary("The password to access the store.")
+  private String password;
```
Comment (marianogonzalez) [inline extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java]: add `@Password`

File: extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ */
+public class WssKeyStoreConfiguration implements WssStoreConfiguration {
+
+  @Parameter
+  @Summary("The alias of the private key to use")
+  private String alias;
+
+  @Parameter
+  @Summary("The password used to access the private key.")
+  private String keyPassword;
```
Comment (marianogonzalez) [inline extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java]: add `@Passowrd`

File: extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.ws.api.security.config;
```
Comment (marianogonzalez) [inline extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java]: does this actually need to be API? Doesn't seem like it to me

File: extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssTrustStoreConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+ */
+public class WssTrustStoreConfiguration implements WssStoreConfiguration {
+
+  @Parameter
+  @Summary("The location of the TrsutStore file")
+  private String trustStorePath;
+
+  @Parameter
+  @Summary("The password to access the store.")
+  private String password;
```
Comment (marianogonzalez) [inline extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssTrustStoreConfiguration.java]: password

File: extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java (hunk 3 above)
Comment (juandesi) [inline extensions/wsc/src/main/java/org/mule/extension/ws/api/security/config/WssKeyStoreConfiguration.java]: why? It must be api, the mule user interacts directly with this element.

**Pull request merged**